lib/taiko.js

```diff
@@ -1644,7 +1644,7 @@ module.exports.link = (attrValuePairs, ...args) => {
 };
 
 /**
- * This {@link selector} lets you identify a list item (HTML <li> element) on a web page with label or attribute and value pairs and proximity selectors.
+ * This {@link selector} lets you identify a list item (HTML element) on a web page with label or attribute and value pairs and proximity selectors.
```

Comment:
This is to specify that the, `listItem` api identifies the html `<li>` tag and that is required.

Reply:
With commit : 51102a49067dd7e7b9af671f071b45c5d435db4e docs should now display `<li>` tag properly.
Closing this for now. Thanks for the PR!

Reply:
Thank you for your quick reply.